extensions/telegram/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -22,7 +22,7 @@
 #
 ---
 name: "Camel Telegram"
-description: "Send and receive messages acting as a Telegram Bot Telegram Bot API"
+description: "Send and receive messages using the Telegram Bot API"
```

Comment:
It might be better to exclude it from this commit? I think we need a seperate commit to regenerate the meata data.

Reply:
this change is included in its own commit commit https://github.com/apache/camel-quarkus/pull/5922/commits/6f8bcbc53804ed51a46859721b284da307e38a45

I can remove it from the PR if required.